File: C++/AccionNota.cpp
#include "AccionNota.h"

AccionNota::AccionNota(const std::string& cedula, float nota)
    : cedula(cedula), nota(nota) {}

std::string AccionNota::getCedula() const {
    return cedula;
}

float AccionNota::getNota() const {
    return nota;
}

File: Java/sga/model/AccionNota.java
package sga.model;

public class AccionNota {
    private String cedula;
    private float nota;

    public AccionNota(String cedula, float nota) {
        this.cedula = cedula;
        this.nota = nota;
    }

    public String getCedula() {
        return cedula;
    }

    public float getNota() {
        return nota;
    }
}